packages/dd-trace/src/profiling/constants.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+const SnapshotKinds = Object.freeze({ Periodic: 'periodic', OnShutdown: 'on_shutdown', OnOutOfMemory: 'on_oom' })
+const OOMExportStrategies = Object.freeze({ Process: 'process',
+  AsyncCallback: 'async',
+  InteruptCallback: 'interrupt',
+  Logs: 'logs' })
```

Comment:
As before, we put each property on a new line in multiline objects. Also, we generally reserve uppercase-first for class names only, properties and variables are always camelCase.

Reply:
Ah, yes, for constants we do [all uppercase](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/constants.js). We also generally destructure the values out of it so there's not really any intermediate name for that, but I think generally speaking we'd consider that a regular variable/property and therefore follow the same camelCase naming. So something like:

```js
const snapshotKinds = Object.freeze({
  PERIODIC: 'periodic',
  ON_SHUTDOWN: 'on_shutdown',
  ON_OUT_OF_MEMORY: 'on_oom'
})
const oomExportStrategies = Object.freeze({
  PROCESS: 'process',
  ASYNC_CALLBACK: 'async',
  INTERUPT_CALLBACK: 'interrupt',
  LOGS: 'logs'
})
```

We have also not generally used `Object.freeze(...)`, but I think it's fine to use here. Can't think of any reason _not_ to use it. 🤷🏻

Reply:
I made the changes